packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -1506,6 +1511,45 @@ describe('e2e', function () {
             ).to.have.lengthOf(1);
           });
         });
+
+        it('writes custom log directly', async function () {
+          const connectionString = await testServer.connectionString();
+          await shell.executeLine(
+            `connect(${JSON.stringify(connectionString)})`
+          );
```

Comment:
What does this do?

Reply:
It was going to check that both commands write to the same log. It is coming :)